charts/kof-collectors/templates/opentelemetry/collector.yaml

```
@@ -37,18 +37,17 @@ spec:
 
     processors:
       batch: {}
-
     exporters:
       debug: {}
       prometheusremotewrite:
         endpoint: {{ .Values.kof.metrics.endpoint }}
-        tls:
-          insecure: true
+{{- include "kof-collectors.helper.tls_options" .Values.kof.metrics | indent 8 }}
```

If you put a `{{-` (removing all indentation from the beginning), then please put a template to the right position for better readability.
```suggestion
        {{- include "kof-collectors.helper.tls_options" .Values.kof.metrics | indent 8 }}
```